tests/fixtures/major-name-modification.json

```diff
@@ -50,10 +52,35 @@
         "type": "Feature",
         "properties": {
           "name": "BANGALURU",
+          "wikidata": "Q40435",
+          "osm:version": 9
```

Comment:
@geohacker Should the version be > 10 to make sure we check just the name modification and version wise things are good here: https://github.com/mapbox/osm-compare/blob/master/comparators/major-name-modification.js#L21